aries_vcx/src/handlers/out_of_band/receiver.rs

```diff
@@ -136,3 +151,135 @@ fn attachment_to_aries_message(attach: &Attachment) -> VcxResult<Option<AriesMes
         )),
     }
 }
+
+pub async fn oob_invitation_to_legacy_did_doc(
```

Comment:
`oob_invitation_to_diddoc ` moved to `aries_vcx/src/protocols/did_exchange/state_machine/requester/request_sent/helpers.rs` because it's only used locally in didexchange requester. Ultimately we'll get rid of it

`oob_invitation_to_legacy_did_doc` moved to `aries_vcx/src/protocols/oob` along with its own private helper   `normalize_keys_as_naked`

Reply:
Yes, we need to get rid of these conversions, sooner the better